numpy/core/shape_base.py

```diff
@@ -275,3 +278,73 @@ def hstack(tup):
         return _nx.concatenate(arrs, 0)
     else:
         return _nx.concatenate(arrs, 1)
+
+def stack(arrays, axis=0):
+    """
+    Join a sequence of arrays along a new axis.
+
+    .. versionadded:: 1.10.0
```

Comment:
`.. versionadded:: 1.10.0` and its preceding blank line should come at the end of the summary ;)